documentation/operations/admin-procedures.html

```diff
@@ -20,8 +20,8 @@ <h2 id="system-status">System status</h2>
   <li>Use <a href="../reference/search-api-reference.html#tracelevel">query tracing</a></li>
   <li>Use <a href="../reference/services.html#clients">feed tracing</a></li>
     <!-- ToDo: this works? must document ... -->
-  <li>Use the <a href="../elastic-vespa.html#cluster-controller">
-    Cluster Controller</a> to track the status of search/storage nodes.</li>
+  <li>Use the <em>cluster controller</em> status page (below)
```

Comment:
An explicit link would probably be better than just "(below)" since it lets us scan for links to replace if the content moves again

Reply:
ok, considered for next pr